emails/2025-01-31.md

```diff
@@ -0,0 +1,23 @@
+This is the weekly CEO update from [DoltHub](https://www.dolthub.com/). I'm Tim, the CEO of DoltHub. 
+
+It's our annual company get together in Santa Monica this weekend. We have offices in Seattle and Santa Monica and a handful of remote folks. It will be nice to see everyone in person, except [Zach](https://www'dolthub.com/team#zach).
```

Comment:
```suggestion
It's our annual company get together in Santa Monica this weekend. We have offices in Seattle and Santa Monica and a handful of remote folks. It will be nice to see everyone in person, except [Zach](https://www.dolthub.com/team#zach).
```